src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-dcgm-integration.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: NVIDIA DCGM integration
+tags:
+  - NVIDIA integration
+  - DCGM integration
+  - New Relic integrations
+metaDescription: Use New Relic infrastructure agent to get a dashboard with DCGM metrics.
+---
+import infrastructureNVIDIADCGMDashboard from 'images/infrastructure_screenshot-full_nvidia-dcgm-dashboard.webp'
+
+Our NVIDIA DCGM integration assists you in monitoring the status of GPUs. This integration leverages our infrastructure agent and the Prometheus remote write integration, which is seamlessly integrated with NVIDIA's SMI utility. It provides you with a pre-built dashboard containing crucial DCGM metrics, including GPU utilization, XID error counts, clock and performance states, temperature, power usage.
+
+<img
+    title="NVIDIA DCGM dashboard"
+    alt="NVIDIA DCGM dashboard"
+    src={infrastructureNVIDIADCGMDashboard}
+/>
+
+<figcaption>
+After you set up our NVIDIA DCGM integration, we give you a dashboard for your DCGM metrics.
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+To get data into New Relic, install our infrastructure agent. Our infrastructure agent collects and ingests data so you can keep track of your DCGM performance.
+
+You can install the infrastructure agent two different ways:
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/).
+
+## Configure DCGM-Exporter
+1. Clone the repository
+```
+git clone https://github.com/NVIDIA/dcgm-exporter
+```
+2. Below command is used to open the `dcgm-exporter` directory present in the cloned repository.
+```
+cd dcgm-exporter
+```
+3. Install necessary binaries using below commands.
+```
+make binary
+```
+
+```
+sudo make install
+```
+4. Start the dcgm-exporter using below command
+```
+dcgm-exporter &
```

Comment:
@RamanaReddy8801 Just to confirm, the `&` is a part of the command?

Reply:
Yes, `&` is an essential element of the command.